src/universal/redux/segmentActions.js

```diff
@@ -1,22 +1,53 @@
 import {EventTypes} from 'redux-segment';
+import {selectProfile} from './profileDuck';
 
 /**
  * Sometimes, we just want to track events in segment.
  */
 
 const SEGMENT_EVENT = '@@segment/EVENT';
 
-export function segmentEvent(event, properties) {
-  return {
-    type: SEGMENT_EVENT,
-    meta: {
-      analytics: {
-        eventType: EventTypes.track,
-        eventPayload: {
-          event,
-          properties
+export function segmentEventTrack(event, properties, options) {
+  return (dispatch, getState) => {
+    const profile = selectProfile(getState());
```

Comment:
@mattkrick fair question on duplication of state!

Looking at the `cashay.entities.User[userId]` after logging in on my machine, my user record has:
   * preferredName
   * email
   * id
   * picture

The auth0 profile gives us more information. But, that's not reason enough to use it. I like using existing state if we can.

What we'd need:
   * We capture identify and profile information *just* after login and on session restore from localstorage
   * We know who the user is (i.e. if we have multiple entities in `cashay.entities.User`, who am I?)
   * We capture user id, avatar image, name, and email

If you can suggest a path to do this, I'd be happy to refactor! Else, I am happy with this solution...